ffmpeg/ffmpeg_test.go

```diff
@@ -1838,3 +1838,62 @@ func TestTranscoder_Clip2(t *testing.T) {
 	assert.Equal(t, 601, res.Encoded[0].Frames)
 	assert.Equal(t, int64(22155264), res.Encoded[0].Pixels)
 }
+
+func TestResolution_Clamp(t *testing.T) {
+	// expect no error
```

Comment:
Nit: In Golang it's very popular to use table driven unit tests. You can consider converting your test in such a way. Anyway, up to you.

Here are some examples:
- https://dev.to/boncheff/table-driven-unit-tests-in-go-407b
- https://yourbasic.org/golang/table-driven-unit-test/